pyproject.toml

```diff
@@ -64,7 +64,7 @@ experimental = [
   "tenacity",
 ]
 llama-index = [
-  "llama-index~=0.9.0",
+  "llama-index~=0.9.0,<=0.9.6.post2"
```

Comment:
```suggestion
  "llama-index<0.9.7"
```

Reply:
we actually have a lower bound requirement because other changes